app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java

```diff
@@ -458,9 +460,10 @@ public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
                                 if(controller != null && controller.canSetPlaybackSpeed()) {
                                     float playbackSpeed = (progress + 10) / 20.0f;
                                     controller.setPlaybackSpeed(playbackSpeed);
-                                    String speed = String.format("%.2f", playbackSpeed);
-                                    UserPreferences.setPlaybackSpeed(speed);
-                                    txtvPlaybackSpeed.setText(speed + "x");
+                                    String speedPref = String.format(Locale.US, "%.2f", playbackSpeed);
```

Comment:
This probably isn't the time to do this (since I should be merging a building a new release), but at some point could you:

1. add a comment on why we have to do this (it's rather unexpected)
2. Couldn't UserPreferences return/set the speed as a float and then users of the interface don't have to mess with this?